web-console/3-project-details/jobs.md

```diff
@@ -0,0 +1,59 @@
+
+
+# Jobs
+
+- A Job creates one or more pods which run to completion, i.e. they don't run continuously like deployments.
+- A Cron Job extends a regular Job by allowing users to schedule when a job will run.
+- This documentation uses "Job" to refer to both Jobs and Cron Jobs, except when Cron Jobs are called out specifically.
+
+## Overview
+- Jobs and Cron Jobs would be listed under a "Jobs" heading in the Application and Resource views of the Overview.
+- The heading over individual Job names would specify if it is a Job or Cron Job.
+- Jobs (and Cron Jobs) that have an app label should be grouped with that application in the Application view.
+
+### Collapsed State
+![jobs overview collapsed](img/jobs-overview-collapsed.png)
+- Jobs are listed on the Overview page similarly to deployments.
+- Currently running jobs have the pod donut donut.
+- For Jobs that are not running, the pod donut is replaced by a "Start Job" button which would initiate a job run.
+- The kebab includes the following actions: "View Logs", "Edit" and "Delete".
+
+
+### Expanded State
+
+#### Running
+![jobs overview expanded running](img/jobs-overview-epxanded-running.png)
+- In the expanded state information about the container would be shown, as with deployments.
+- The pod donut is shown and the user is able to scale up and down.
+- There is a separate section for "Job Runs"
+- Run number and status are shown on the left.
+- Status for completed jobs includes the duration.
+- Logs for a running pod are shown in this view similar to builds, with a link to the full logs.
```

Comment:
If there are multiple pods (or multiple containers in a pod), we can only show the logs for one.

Reply:
I would assume that we'd just pick one. Like we do for replication controllers.